async_test.go

```diff
@@ -196,3 +196,63 @@ func TestLongRunningAsyncQuery(t *testing.T) {
 		}
 	}
 }
+
+func runLongRunningAsyncQuery(ctx context.Context, t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		_ = dbt.mustQueryContext(ctx, "CALL SYSTEM$WAIT(100, 'SECONDS')")
+	})
+}
+
+func TestLongRunningAsyncQueryFetchResultByID(t *testing.T) {
+	queryIDChan := make(chan string, 1)
+	ctx := WithAsyncMode(context.Background())
+	ctx = WithQueryIDChan(ctx, queryIDChan)
+
+	goRoutineChan := make(chan string)
+	go func(grCh chan string, qIDch chan string) {
```

Comment:
Not sure, why do we need this code? Why can't we just use `queryID := <-queryIDChan` below?

Reply:
You're right, I don't need the go func.